runtime/doc/lsp.txt

```diff
@@ -243,59 +239,24 @@ For |lsp-response|, each |lsp-handler| has this signature: >
                                     current document version to check if the
                                     response is "stale".
                                     See also |b:changedtick|.
+                                    For |lsp-notification|: omitted.
         {config}    (table)
-                        Configuration for the handler.
-
-                        Each handler can define its own configuration table
-                        that allows users to customize the behavior of a
-                        particular handler.
-
-                        To configure a particular |lsp-handler|, see:
-                            |lsp-handler-configuration|
-
-
-    Returns: ~
-        The |lsp-handler| can respond by returning two values: `result, err`
-        Where `err` must be shaped like an RPC error:
-            `{ code, message, data? }`
-
-        You can use |vim.lsp.rpc.rpc_response_error()| to create this object.
-
-For |lsp-notification|, each |lsp-handler| has this signature: >
```

Comment:
Notification handler is very similar to the request case, so it's probably more helpful to combine them than split them.

Reply:
> The method also doesn't take a handler:

Yeah I was wondering about that :) So the doc should just be removed, right?

What is implied by these tests?: https://github.com/neovim/neovim/blob/e27377e33e029a9c2efb282d4c4e51effd76dd88/test/functional/plugin/lsp_spec.lua#L281

That's just testing for the message, not a handler of any sort?

Reply:
> So the doc should just be removed, right?

Yes

> That's just testing for the message, not a handler of any sort?

It's only testing that the notification sent by the client arrives at the server.